configs/test-utils/src/ThrowError.tsx

```diff
@@ -14,7 +24,7 @@ export const ThrowError = ({ message, after = 0, children }: ThrowErrorProps) =>
   if (isNeedError) {
     throw new Error(message)
   }
-  useSetTimeout(() => setIsNeedError(true), after)
+  useTimeout(() => setIsNeedError(true), after)
   return <>{children}</>
```

Comment:
is this need for wrapping Fragment component?

Reply:
![image](https://github.com/suspensive/react/assets/61593290/66251d99-848e-49ec-90ce-d8188efb4507)